apps/voting/app/src/components/SummaryRows.js

```diff
@@ -70,6 +74,17 @@ function SummaryRow({ color, label, pct, token }) {
   )
 }
 
+const You = () => (
```

Comment:
Perhaps we should extract this out into its own component since it's used in a few places?

Reply:
`aragonUI`? or local?

Reply:
Already there :-)

```jsx
<Tag size="small">you</Tag>
```

Reply:
I'll add one locally, later we can decide if we want one for `aragonUI`

Reply:
Update: added one locally before I saw your comment @bpierre - I'll update